### ISSUE TITLE
I have a question

### ISSUE THREAD

**Issue by woowonjin** [Megvii-BaseDetection/YOLOX#1099]
There's "pad = (ksize-1) // 2" in /yolox/models/network_blocks.py 37 lines
I think if you want to add padding "same" you can use 
self.conv = nn.Conv2d(
            in_channels,
            out_channels,
            kernel_size=ksize,
            stride=stride,
            padding= "same",
            groups=groups,
            bias=bias,
        )
I think if you use like "pad = (ksize - 1) // 2", it will not be same if the stride is not 1.
I want to know your idea.

**Comment by FateScript** [comment]
We did not using pad="same" very often. And for ksize = 3, 1, `pad = (ksize-1) // 2` is enough.

**Comment by woowonjin** [comment]
@FateScript 
Did you mean that you don't care about the cases when the stride over the 1?
I mean that when the stride is changed (e.g. stride = 2), '''pad = (ksize-1) // 2''' can't change the output_size to same with input_size

**Comment by FateScript** [comment]
For pytorch Conv2d, padding="same" also doesn't work. You could find the following from documentation: `padding='valid' is the same as no padding. padding='same' pads the input so the output has the shape as the input. However, this mode doesn’t support any stride values other than 1.`

**Comment by woowonjin** [comment]
Thank you.!!
